app/assets/stylesheets/_footer.scss

```diff
@@ -0,0 +1,44 @@
+footer {
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
These 3 styles would only apply on position absolute or fixed elements. Is this element one of those? If not, these 3 won't actually do anything